meta-intel-extras/recipes-support/grub/grub-efi/grubenv-copy.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+if [ ! -d /boot/EFI/BOOT/ ]; then
```

Comment:
isn't it excessive?

Reply:
This is needed because when the B partition is booted after swupdate, the boot partition is not mounted on it. I am not sure why but that is the reason to do this.